Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -31,6 +31,35 @@ function Format-Splat {
     "$parameters" # String join the array.
 }
 
+function Generate-TlsError {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)] $exception 
+    )
+
+    if ($exception -eq $null)
+    {
+        return
+    }
+
+    $isWebException = $true
+    $innerException = $exception
+    while ($innerException.GetType() -ne [System.Net.WebException])
+    {
+        $innerException = $innerException.InnerException
+        if ($innerException -eq $null)
+        {
+            $isWebException = $false
+            break
+        }
+    }
+
+    if (($isWebException -eq $true) -and ($innerException.InnerException -ne $null) -and ($innerException.InnerException.GetType() -eq [System.IO.IOException]))
+    {
+        Write-VstsTaskError -Message (Get-VstsLocString -Key AZ_UnsupportedTLSError)
```

Comment:
Print actual exception as well

Reply:
This function only prints the TLS error. The original exception handling is not changed. Original exceptions are still thrown.